Source/WebKit/UIProcess/UserMediaPermissionRequestManagerProxy.cpp

```diff
@@ -122,6 +122,16 @@ UserMediaPermissionRequestManagerProxy::~UserMediaPermissionRequestManagerProxy(
     invalidatePendingRequests();
 }
 
+void UserMediaPermissionRequestManagerProxy::ref() const
+{
+    m_page->ref();
```

Comment:
Refing the page does not guarantee that its UserMediaPermissionRequestManagerProxy member will not be set to null (this happens when the page is reset).
It is probably safer to make it RefCounted.

Reply:
Oh, I missed that. Will fix.